inc/sairouterintf.h

```diff
@@ -46,7 +46,10 @@ typedef enum _sai_router_interface_type_t
     SAI_ROUTER_INTERFACE_TYPE_PORT,
 
     /** VLAN Router Interface Type */
-    SAI_ROUTER_INTERFACE_TYPE_VLAN
+    SAI_ROUTER_INTERFACE_TYPE_VLAN,
+
+    /** Loopback Router Interface Type */
+    SAI_ROUTER_INTERFACE_TYPE_LOOPBACK
```

Comment:
In the sai_router_interface_attr_t, what are the valid attributes for this LOOPBACK router interface?